connect_bi_reporter/feeds/api/views.py

```diff
@@ -0,0 +1,43 @@
+from logging import Logger
+
+from connect.client import ClientError, ConnectClient
+from fastapi import Depends, Request, status
+from connect.eaas.core.decorators import router
+from connect.eaas.core.inject.common import get_logger
+from connect.eaas.core.inject.synchronous import get_installation, get_installation_client
+
+from connect_bi_reporter.connect_services.reports import get_report_schedule
+from connect_bi_reporter.db import get_db, VerboseBaseSession
+from connect_bi_reporter.feeds.api.schemas import FeedCreateSchema, FeedSchema, map_to_feed_schema
+from connect_bi_reporter.feeds.errors import FeedError
+from connect_bi_reporter.feeds.services import create_feed, get_credential_or_404
+from connect_bi_reporter.utils import get_user_data_from_auth_token
+
+
+class FeedsWebAppMixin:
+
+    @router.post(
+        '/feeds',
+        summary='Creates a new Feed',
+        response_model=FeedSchema,
+        status_code=status.HTTP_201_CREATED,
+    )
+    def create_feed(
+        self,
+        feed_schema: FeedCreateSchema,
+        db: VerboseBaseSession = Depends(get_db),
+        client: ConnectClient = Depends(get_installation_client),
+        installation: dict = Depends(get_installation),
+        logger: Logger = Depends(get_logger),
+        request: Request = None,
+    ):
+        get_credential_or_404(db, installation, feed_schema.credential.id)
```

Comment:
[COD-C] IMHO this should be also a 400, as we are validating form data, like with the schedule.

Reply:
400 is better I agree :)

Reply:
updated!